src/Ztm.Zcoin.NBitcoin/Exodus/PropertyAmount.cs

```diff
@@ -132,5 +132,15 @@ public static bool operator>(PropertyAmount first, PropertyAmount second)
         {
             return first.value >= second.value;
         }
+
+        public static PropertyAmount Negate(PropertyAmount amount)
```

Comment:
We should overload `-` operator instead so the caller site just do `-amount`.

Reply:
Added